paddle/fluid/framework/details/build_strategy.h

```diff
@@ -74,8 +74,12 @@ struct BuildStrategy {
 
   bool fuse_elewise_add_act_ops_{false};
 
+  bool fuse_all_optimizer_ops_{false};
```

Comment:
enable by default?

Reply:
?

Reply:
I'm not sure if this pass can improve performance on all models, today I'll turn it on and run on CE.

Reply:
what's the result?

Reply:
For some models, the speed can be increased by 19%, for example, the model_object_detection model (in the case of 4 cards), in some models, the speed decreases slightly by 1%.